pom.xml

```diff
@@ -163,5 +191,24 @@
 				</dependency>
 			</dependencies>
 		</profile>
+		<profile>
+			<id>java17</id>
+			<activation>
+				<jdk>17</jdk>
+			</activation>
+			<build>
+				<plugins>
+					<plugin>
+						<groupId>org.apache.maven.plugins</groupId>
+						<artifactId>maven-compiler-plugin</artifactId>
+						<configuration>
+							<source>17</source>
+							<target>17</target>
+						</configuration>
+					</plugin>
+				</plugins>
+			</build>
+		</profile>
```

Comment:
why adding a profile which changes nothing ?

Reply:
Indeed. It is an unnecessary extension of pom file.